Documentation/Blazorise.Docs/Pages/Docs/Components/Typographies/TypographyPage.razor

```diff
@@ -69,6 +69,51 @@
     <DocsPageSectionSource Code="TypographyDisplayHeadingExample" />
 </DocsPageSection>
 
+<DocsPageSection>
+    <DocsPageSectionHeader Title="Unordered Lists">
+        Use the <Code>UnorderedList</Code> component to show an unordered or ordered list of items based on multiple styles, layouts, and variants.
+    </DocsPageSectionHeader>
+    <DocsPageSectionContent Outlined FullWidth>
+        <TypographyUnorderedListExample />
+    </DocsPageSectionContent>
+    <DocsPageSectionSource Code="TypographyUnorderedListExample" />
+</DocsPageSection>
+
+<DocsPageSection>
+    <DocsPageSectionHeader Title="Ordered Lists">
+        Use the <Code>OrderedList</Code> component to show an ordered list of items based on multiple styles, layouts, and variants.
+    </DocsPageSectionHeader>
+    <DocsPageSectionContent Outlined FullWidth>
+        <TypographyOrderedListExample />
+    </DocsPageSectionContent>
+    <DocsPageSectionSource Code="TypographyOrderedListExample" />
+</DocsPageSection>
+
+<DocsPageSection>
+    <DocsPageSectionHeader Title="List Style Image">
+        <Paragraph>
+            Control the marker image for list items using the <Code>ListStyleImage</Code> parameter.
+        </Paragraph>
+        <Paragraph>
+            This parameter accepts either a <Strong>Base64</Strong> encoded string that represents an image, or a <Strong>URL</Strong> that points to an image resource.
+        </Paragraph>
+    </DocsPageSectionHeader>
+    <DocsPageSectionContent Outlined FullWidth>
+        <TypographyListImageStyleExample />
+    </DocsPageSectionContent>
+    <DocsPageSectionSource Code="TypographyListImageStyleExample" />
+</DocsPageSection>
+
+<DocsPageSection>
+    <DocsPageSectionHeader Title="Unstyled List">
+        Use the <Code>Unstyled</Code> parameter to disable the list style bullets or numbers.
```

Comment:
What if both **Unstyled** and **ListStyleImage** are defined?

Reply:
Well, it won't work because Unstyled is hiding the bullet points.